docs/src/main/asciidoc/picocli.adoc

```diff
@@ -88,11 +91,71 @@ public class EntryCommand {
     public void setName(String name) {
         this.name = name;
     }
-
 }
 ----
 
-== Command line application with multiple Commands
+=== Packaging
```

Comment:
I'm not sure I would put this section here. It is very specific. I would rather keep the existing structure and add a `Packaging hints` paragraph later.

Reply:
I would like to disagree on this on. So far the article has basically not much structure, since all headlines are on the same level. I want to make it more similar to the other guides and provide a structure with a basic example that cover the happy path or 80% case or whatever one might wanna call it. Afterwards, more advanced solutions are outlined.

Having 3 different ways to implement the entry points, is not something a first time reader can/wants to deal with IMHO.